lightyear/src/server/replication.rs

```diff
@@ -6,6 +6,13 @@ use crate::server::connection::ConnectionManager;
 use crate::server::prediction::compute_hash;
 use crate::shared::replication::plugin::ReplicationPlugin;
 
+/// Configuration related to replicating the server's World to clients
+#[derive(Clone, Default, Debug)]
+pub struct ReplicationConfig {
+    /// Set to true to disable replicating this server's entities to clients
+    pub disable: bool,
```

Comment:
Here it's called `disable`, and in the other one it's called `enabled`

Reply:
That's on purpose; I think 99% of cases are only interested in server->client replication. Client->server replication is a bit more niche; and will need further work. I probable need to create a new component or field `replication_direction`